modular_exponential/modular_exponential.py

```diff
@@ -0,0 +1,21 @@
+def mod_exponent(base, power, mod):
+    res = 1  # Initialize result
```

Comment:
As from the names it is preety much clear about the variable use, but i added the usage of the function.